cmd/internal/kube/daemonset/kube_events_worker.go

```diff
@@ -9,6 +9,12 @@ import (
 	coreV1 "k8s.io/api/core/v1"
 )
 
+const (
+	allRequiredEnvVarsAbsentMsg    = "All required environment variables are absent."
```

Comment:
Non-blocking, you can make the error itself the variable instead of just the string, like this:
https://pkg.go.dev/errors#pkg-variables